eva_cttv_pipeline/trait.py

```diff
@@ -10,30 +10,33 @@ def map_efo(trait_2_efo_dict, name_list):
     that id.
     :param name_list: List of clinvar trait names for the trait, with the preferred trait name (if
     one exists) in the first element.
-    :return: The clinvar name, ontology id, and ontology label, for the clinvar name in the
+    :return: The clinvar name, and mapping to an ontology id and term, for the clinvar name in the
     name_list with the lowest element that is in the trait_2_efo_dict. If none of the names in the
     name_list are keys in the trait_2_efo_dict then None is returned in each position.
     """
-    trait_string = name_list[0].lower()  # Try first element, which should be preferred name if it exists
-    if trait_string in trait_2_efo_dict:
-        ontology_id = trait_2_efo_dict[trait_string][0]
-        ontology_label = trait_2_efo_dict[trait_string][1]
-        return trait_string, ontology_id, ontology_label
-    else:
-        # Otherwise cycle through the list, the first name that is in the dict
-        # is returned along with the ontology id and label from the dict
-        for trait in name_list[1:]:
-            trait_string = trait.lower()
-            if trait_string in trait_2_efo_dict:
-                ontology_id = trait_2_efo_dict[trait_string][0]
-                ontology_label = trait_2_efo_dict[trait_string][1]
-                return trait_string, ontology_id, ontology_label
+    for trait in name_list:
+        trait_string = trait.lower()
+        if trait_string in trait_2_efo_dict:
+            return trait_string, trait_2_efo_dict[trait_string]
 
-    return None, None, None  # If none of the names in the list are keys in the dict then None is returned
+    return name_list[0].lower(), None  # If none of the names in the list are keys in the dict then None is returned
```

Comment:
That's true, looking at its only usage (https://github.com/tomdcsmith/eva-cttv-pipeline/blob/d4fb27a5bf14d15859c559c130f8a66fb547faf7/eva_cttv_pipeline/clinvar_to_evidence_strings.py#L318) it doesn't matter whether it's None or trait_string since it's not used if there's no mapping. To be consistent I'll change it back.